app/assets/javascripts/discourse/app/templates/topic.hbs

```diff
@@ -44,9 +44,9 @@
 
             {{plugin-outlet name="edit-topic" args=(hash model=model buffered=buffered)}}
 
-            <div class="edit-controls">
-              {{d-button action=(action "finishedEditingTopic") class="btn-primary submit-edit" icon="check"}}
-              {{d-button action=(action "cancelEditingTopic") class="btn-default cancel-edit" icon="times"}}
+            <div class="edit-controls" aria-label="helo">
```

Comment:
"helo"?

Reply:
Wow! I must have reverted an edit from when testing this in the browser.